packages/js/email-editor/src/store/resolvers.ts

```diff
@@ -27,8 +29,13 @@ export function* getPersonalizationTagsList() {
 	yield setIsFetchingPersonalizationTags( true );
 
 	try {
+		const pathWithQuery = addQueryArgs(
+			'/woocommerce-email-editor/v1/get_personalization_tags',
+			{ postType, context: 'edit' }
+		);
```

Comment:
When testing locally I noticed that we filter the personalization tags already on the server. The thing is that in some integration cases (e.g., WP new admin) the single instance of the editor might be used with various post types.
My concern here is that if we open the editor with `woo_email` and we run this resolver, we get only personalization tags for `woo_email`, then we pass the switch to `mailpoet_email`, and I'm not sure this resolver will run again.

Can we make sure the resolvers run again if we dynamically change the post type?  Or perhaps could we fetch app PTs from the server and filter only on the client?